definitions/ext-ibmmqmanager/definition.yml

```diff
@@ -0,0 +1,29 @@
+domain: EXT
+type: IBMMQQUEUEMANAGER
+
+synthesis:
+  name: qManagerName
+  identifier: qManagerName
+  conditions:
+    - attribute: eventType
+      value: "MQObjectStatusSample"
+
+tags:
```

Comment:
The tags section needs to be under synthesis (it belong to the synthesis rule provided)